pandas/core/window/rolling.py

```diff
@@ -1459,8 +1460,24 @@ def median(
         window_func = window_aggregations.roll_median_c
         return self._apply(window_func, name="median", **kwargs)
 
-    def std(self, ddof: int = 1, *args, **kwargs):
+    def std(
+        self,
+        ddof: int = 1,
+        *args,
+        engine: str | None = None,
+        engine_kwargs: dict[str, bool] | None = None,
+        **kwargs,
+    ):
         nv.validate_window_func("std", args, kwargs)
+        if maybe_use_numba(engine):
+            if self.method == "table":
+                raise NotImplementedError("std not supported with method='table'")
```

Comment:
hit this in tests?

Reply:
Yup here: https://github.com/pandas-dev/pandas/pull/44461/files#diff-4fea2b6ab86ef8bb3c962ab4ee5667c2ea4df13bfab32ffcf03ec130451699dcR296